src/main/webapp/app/overview/course-conversations/posting-summary/posting-summary.component.html

```diff
@@ -0,0 +1,93 @@
+<div class="posting-summary rounded p-2" (click)="onTriggerNavigateToPost()">
+    @if (post() !== undefined) {
```

Comment:
Can we use use shorter non-null check for this or does yours entail more strict rules? 
```suggestion
    @if (post()) {
```

Reply:
Okay I could see how this might be problematic with the Signal.. Still it would be nice if the repetitive null-checks could be shortened

Reply:
I tried to make the template file more readable by introducing booleans "isShowContent", "isShowHeader", which get evaluated in the effect({}) function. 👍